### DISCUSSION TITLE
kontan.co.id - Ads Script

### DISCUSSION THREAD

**Discussion by Recehan-Slayer** [realodix/AdBlockID#729] [General]
**Jenis iklan** (hapus yang tidak perlu)
- Lainnya

**Sample URL**
https://kontan.co.id/

**Penjelasan Lebih Lanjut**
`https://dalj98ktasppa.cloudfront.net/47f747cab855ab2ecaa51f88cc65d420`
`https://assets.kontan.co.id/min/b=live/general/js&f=script.general.js,script.home.js,ads.js`
itu 3p ama 1p scrips ads iya ga gan\?

**Comment by realodix** [comment]
> dalj98ktasppa.cloudfront.net/47f747cab855ab2ecaa51f88cc65d420

Kalau dibongkar, ada beberapa kode iklan yang ditrigger dari assets.kontan.co.id

> https://assets.kontan.co.id/min/b=live/general/js&f=script.general.js,script.home.js,ads.js

Saya gak ketemu link di atas, versi gabungan file JS dan diminify. Link file asli JS untuk iklan https://assets.kontan.co.id/live/general/js/ads.js dan sudah terblokir oleh easylist dengan `/js/ads.`

**Comment by Recehan-Slayer** [comment]
sama ini gan itu tracker bukan
`https://dhpikd1t89arn.cloudfront.net/js/include/sharethefacts-v1.js`
dari https://www.tempo.co/

**Comment by Recehan-Slayer** [comment]
sama ini gan dr https://uzone.id/
<details>

![Untitled](https://user-images.githubusercontent.com/9379770/73646372-e0050a00-46ab-11ea-9173-5ef46b207ed8.png)

**Comment by realodix** [comment]
> sama ini gan itu tracker bukan
> `https://dhpikd1t89arn.cloudfront.net/js/include/sharethefacts-v1.js`
> dari https://www.tempo.co/

Ada di [easylist/easylist/fanboy-addon/fanboy_annoyance_thirdparty.txt#L87](https://github.com/easylist/easylist/blob/bd9934bf73ca42613659006fe5bdff000a71a447/fanboy-addon/fanboy_annoyance_thirdparty.txt#L87)
